src/components/Loading.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const Loading: React.FC = () => <StyledLoading />
```

Comment:
I don't understand why this is here.

Reply:
you mean the decleration?

Reply:
Yes, why not use the styled component directly?

Reply:
Done